docs/DevelopmentSetupAndroid.md

```diff
@@ -11,7 +11,7 @@ This guide describes basic steps of the Android development environment setup th
 
 ### Install Git
 
-  - **On Mac**, if you have installed [XCode](https://developer.apple.com/xcode/), Git is already installed, otherwise run the following:
+  - **On Mac**, Install [XCode](https://developer.apple.com/xcode/). If Git is not already installed, run the following:
```

Comment:
I think it was better previously. XCode is not required for Android development. I was just mentioned because if someone has XCode, he already has Git, and no need to do anyting..